Applications/HexEditor/HexEditor.cpp

```diff
@@ -44,6 +44,20 @@ void HexEditor::set_buffer(const ByteBuffer& buffer)
     update_status();
 }
 
+void HexEditor::fill_selection(u8 fill_byte)
+{
+    if (m_selection_start == -1 || m_selection_end == -1 || (m_selection_end - m_selection_start) < 0 || m_buffer.is_empty())
```

Comment:
It would be nice to have something like a `bool has_selection() const` that we could ask instead of writing this long check out every time :)

Reply:
I was considering it before, should have went with my gut instinct and did it originally. :-P